core/src/main/scala/io/finch/request/package.scala

```diff
@@ -162,6 +180,8 @@ package object request {
       def apply[Req](req: Req)(implicit ev: Req => HttpRequest) =
         try number.toFuture
         catch { case _: IllegalArgumentException => ValidationFailed(param, rule).toFutureException }
+
+      override def toString: String = s"param: $param"
```

Comment:
In case it was too vague: I'd really like to leave an open door for the following:

```scala
val bePositive = ValidationRule[Int]("be positive"){ _ > 0}

for { 
  a <- RequiredIntParam("a").should(bePositive)
  b <- RequiredIntParam("b").should(bePositive)
  c <- RequiredIntParam("c").should(bePositive)
} yield (a, b, c)
```

I'm not sure I'd enjoy writing this out every time. And for allowing such a syntax, the param name cannot be in the rule.